routers/admin/users.go

```diff
@@ -243,7 +244,20 @@ func EditUserPost(ctx *context.Context, form auth.AdminEditUserForm) {
 	u.AllowGitHook = form.AllowGitHook
 	u.AllowImportLocal = form.AllowImportLocal
 	u.AllowCreateOrganization = form.AllowCreateOrganization
-	u.ProhibitLogin = form.ProhibitLogin
+
+	var doerID int64
```

Comment:
pleace delete line 248 to 254 use `ctx.User.ID`

Reply:
do it , Thanks for your suggestion. I forget this verb before